modular_skyrat/modules/emotes/code/additionalemotes/overlay_emote.dm

```diff
@@ -0,0 +1,74 @@
+/datum/emote
+	var/overlay_emote = 'modular_skyrat/master_files/icons/effects/overlay_effects.dmi'
+
+/datum/emote/living/sweatdrop
+	key = "sweatdrop"
+	key_third_person = "sweatdrops"
+
+/datum/emote/living/sweatdrop/run_emote(mob/living/user, params, type_override, intentional)
+	. = ..()
+	var/mutable_appearance/I = mutable_appearance(overlay_emote, "sweatdrop", ABOVE_MOB_LAYER)
```

Comment:
Use descriptive variable names

Reply:
I've done so, but I assumed that the short letter stuff has an original meaning that always makes sense. I is for image, O is for object, H is for human, S is for silicon, etc. I'm sure I have more one letter variables because of this, should I go through those too?

Reply:
Nah short letter stuff is because past coders were shit and didn't use descriptive variables, and no one barring insane people (myself) want to clean them up